src/common/components/grid/grid.tsx

```diff
@@ -40,6 +40,7 @@ export interface IGridRowDataDetailsExt {
   fullWidth: boolean;
   isVisible: boolean;
 };
+export interface GridRowNode extends RowNode {};
```

Comment:
do you think IGridRowNode is better (add the I before)

Reply:
Aaaaa... those, I keep them as much as close(naming convention) to original one(s)[just prefix them with "**Grid**"] there is a lot of such "interfaces".

Let's leave it this way.